### ISSUE TITLE
caffe batch_norm的转换

### ISSUE THREAD

**Issue by guispor7** [XiaoMi/mace#467]
对于batch_norm layer ,目前代码里面只支持
batch_norm_param {
        use_global_stats: true
    }
这种case，当use_global_stats: false的时候，要重新计算mean 和 variance，比如resnet50的就会出现这样的参数设置。
请问要怎么实现？
谢谢

**Comment by lee-bin** [comment]
可以参考下https://caffe.berkeleyvision.org/tutorial/layers/batchnorm.html
https://github.com/KaimingHe/deep-residual-networks/blob/master/prototxt/ResNet-50-deploy.prototxt
一般来说在训练时`use_global_stats`会设置为false，而推理时`use_global_stats`设置为true
